com.valvesoftware.Steam.metainfo.xml

```diff
@@ -73,33 +73,10 @@
     <release version="1.0.0.62" date="2020-03-19"/>
     <release version="1.0.0.61" date="2019-04-18"/>
   </releases>
-    <content_rating type="oars-1.1">
-    <content_attribute id="violence-cartoon">intense</content_attribute>
-    <content_attribute id="violence-fantasy">intense</content_attribute>
-    <content_attribute id="violence-realistic">intense</content_attribute>
-    <content_attribute id="violence-bloodshed">intense</content_attribute>
-    <content_attribute id="violence-sexual">intense</content_attribute>
-    <content_attribute id="violence-desecration">intense</content_attribute>
-    <content_attribute id="violence-slavery">intense</content_attribute>
-    <content_attribute id="violence-worship">intense</content_attribute>
-    <content_attribute id="drugs-alcohol">moderate</content_attribute>
-    <content_attribute id="drugs-narcotics">moderate</content_attribute>
-    <content_attribute id="drugs-tobacco">moderate</content_attribute>
-    <content_attribute id="sex-nudity">intense</content_attribute>
-    <content_attribute id="sex-themes">intense</content_attribute>
-    <content_attribute id="sex-homosexuality">moderate</content_attribute>
-    <content_attribute id="sex-prostitution">moderate</content_attribute>
-    <content_attribute id="sex-adultery">moderate</content_attribute>
-    <content_attribute id="sex-appearance">intense</content_attribute>
-    <content_attribute id="language-profanity">intense</content_attribute>
-    <content_attribute id="language-humor">intense</content_attribute>
-    <content_attribute id="language-discrimination">intense</content_attribute>
+  <content_rating type="oars-1.1">
     <content_attribute id="social-chat">intense</content_attribute>
-    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-info">intense</content_attribute>
     <content_attribute id="social-audio">intense</content_attribute>
-    <content_attribute id="social-location">none</content_attribute>
```

Comment:
Steam allows sharing location up to a country, if I'm not mistaken. So this attribute should be at least `mild`, I guess? Not sure if `mild` value is implemented for `social-location`.

Reply:
GeoIP based location is explicitly excluded.

Reply:
> Steam allows sharing location up to a country, if I'm not mistaken. So this attribute should be at least `mild`, I guess? Not sure if `mild` value is implemented for `social-location`.

Mild is not implemented, and Steam allows you to share your location up to the city level. Since it's not an exact address I elected to mark that as none, but that may be wrong.